myuw/templates/calendar.html

```diff
@@ -9,7 +9,7 @@
 
 {% block content %}
 
-    <div id="main-content" role="main" aria-labelledby="main"></div>
+    <div id="main-content" role="main" aria-labelledby="main-content-label"></div>
```

Comment:
this main-content-label is on div whereas the Academics page is on h2?

Reply:
Yup, this basically means that the contents of this div is labelled by the h2 being referenced. So in this case, the main section of the page has a heading of "Academic Calendar"